### PR TITLE
[pytket-quantinuum] WASM handling and tests for classical operations

### PR DESCRIPTION
**Pull request by ss2165**

### CODE REVIEW

File: modules/pytket-quantinuum/tests/backend_test.py [hunk 1]
```diff
@@ -519,3 +530,22 @@ def test_custom_api_handler(device_name: str) -> None:
...
+
+
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
+@pytest.mark.parametrize(
+    "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
+)
+def test_wasm(
+    authenticated_quum_backend: QuantinuumBackend,
+) -> None:
+    wasfile = WasmFileHandler(str(Path(__file__).parent / "sample_wasm.wasm"))
```
Comment (cqc-melf) [inline modules/pytket-quantinuum/tests/backend_test.py:542]: Have you created a new file or have you used the other one I have uploaded?

File: modules/pytket-quantinuum/setup.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.2",
+        "pytket == 1.3.0rc1",
```
Comment (cqc-melf) [inline modules/pytket-quantinuum/setup.py:41]: Should we change the current version of the extension of the extension to RC, too?

File: modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py [hunk 3]
```diff
@@ -273,7 +276,7 @@ def rebase_pass(self) -> BasePass:
 
     def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
         assert optimisation_level in range(3)
-        passlist = [DecomposeClassicalExp(), DecomposeBoxes()]
+        passlist = [DecomposeBoxes()]
```
Comment (cqc-melf) [inline modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py:279]: Why is this removed?

File: modules/pytket-quantinuum/tests/backend_test.py (hunk 1 above)
Comment (ss2165) [reply modules/pytket-quantinuum/tests/backend_test.py:542]: it's a new file, with some working compiled wasm

**Review by cqc-melf — APPROVED** [review]

**Review by cqc-melf — APPROVED** [review]
